web/custom-plugins/forecast-of-response/src/components/Operations/PhaseView.tsx

```diff
@@ -0,0 +1,202 @@
+          <Table.Body>
+            {Object.entries(defaultVars).map(([key, defaultVal]) => (
+              <Table.Row key={key}>
+                <Table.Cell>{key}</Table.Cell>
+                <Table.Cell>{defaultVal}</Table.Cell>
+                <Table.Cell>
+                  <Input
+                    placeholder={defaultVal}
+                    value={variables[key]}
+                    onChange={(event: Event) =>
+                      setVariables({
+                        ...variables,
+                        [key]: event.target.value,
+                      })
+                    }
+                  />
+                </Table.Cell>
+              </Table.Row>
+            ))}
+          </Table.Body>
+        </Table>
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'space-around',
+            margin: '10px',
+          }}
+        >
+          <Button
+            variant="outlined"
+            color="danger"
+            onClick={() => setVariables(defaultVars)}
+          >
+            Reset
+          </Button>
+          <Button>Create</Button>
+        </div>
+      </Wrapper>
+    </>
+  )
+}
+
+function Result(props: { result: any }) {
+  const { result } = props
+  return <pre>{JSON.stringify(result, null, 2)}</pre>
+}
+
+function SingleSimulationConfig(props: {
+  simulationConfig: TSimulationConfig
+}) {
+  const { simulationConfig } = props
+  const [selectedSim, setSelectedSim] = useState<number>(1)
+  return (
+    <div>
+      <Wrapper>
+        <h3 style={{ margin: '10px' }}>{simulationConfig.name}</h3>
+        <div style={{ display: 'flex' }}>
+          <Table>
+            <Table.Head sticky>
+              <Table.Row>
+                <Table.Cell>SIMA Variable</Table.Cell>
+                <Table.Cell>Value</Table.Cell>
+              </Table.Row>
+            </Table.Head>
+            <Table.Body>
+              {simulationConfig.variables.map(
+                (variableString: string, index) => {
+                  const [key, value] = variableString.split('=')
+                  return (
+                    <Table.Row key={index}>
+                      <Table.Cell>{key}</Table.Cell>
+                      <Table.Cell>{value}</Table.Cell>
+                    </Table.Row>
+                  )
+                }
+              )}
+            </Table.Body>
+          </Table>
+          <ResultWrapper>
+            {!simulationConfig.simulations.length && (
+              <>No simulations has been run</>
+            )}
+            {simulationConfig.simulations.map(
+              (simulation: TSimulation, index: number) => (
+                <SimulationRow
+                  key={index}
+                  even={index % 2 === 0}
+                  selected={index === selectedSim}
+                  onClick={() => setSelectedSim(index)}
+                >
+                  <div>{new Date(simulation.started).toLocaleString()}</div>
```

Comment:
maybe we can use the 'en-GB' locale here

Reply:
fixed. Used "navigator.local"